packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md

```diff
@@ -43,7 +43,7 @@ However, when you disable the feature flag, your old customized templates will r
 
 ##### EA feature requirements
 
-To access this self-serviced EA feature, you need to enable `ENHANCED_EMAIL_MACROS`in the feature flag. For free orgs, you can contact Okta support (`support@okta.com`) to request email customization.
+To access this self-serviced EA feature, you need to enable `ENHANCED_EMAIL_MACROS`in the feature flag. For free orgs, you will need to first contact Okta support (`support@okta.com`) to request email customization.
```

Comment:
Jakub said we never documented the actual feature flag in our docs. If the end user admin can set it maybe send them to this Okta link: https://help.okta.com/en/prod/Content/Topics/Security/manage-EA-and-beta-features.htm or simply mention to contact Okta support to enable the feature.

Reply:
Thanks, there's 2 FFs mentioned here. ENHANCED_EMAIL_MACROS which is full self-service EA via Admin Console, and the email customization flag which we cannot name and must be turned on by support after Security team evaluates the request for risk.